script/generatetypescriptfile.py

```diff
@@ -0,0 +1,166 @@
+import openspace
+import asyncio
+import re
+import shutil
+
+StaticFileName = "openspace-api-static.txt"
+outFileName = "../declaration/openspace-api-js.d.ts"
+
+def writeFunctionDocumentation(file, fun):
```

Comment:
Seem we have settled on a C++ -> 4 whitespace indentation, everything else -> 2 whitespace indentation.  Would think that we can apply that here as well?